config/webpack/loaders.js

```diff
@@ -38,7 +38,8 @@ module.exports = [
           loader: 'postcss-loader',
           options: {
             sourceMap: true,
-          },
+            plugins: () => [require('autoprefixer')]
```

Comment:
This might not be necessary. Check it and delete.

Reply:
This is actually needed, if deleted i get following error when run `bin/webpack`
```
ERROR in ./node_modules/css-loader?{"minimize":false}!./node_modules/postcss-loader?{"sourceMap":true}!./node_modules/resolve-url-loader!./node_modules/sass-loader/lib/loader.js?{"sourceMap":true}!../react-ui-components/dist/tagging.css
Module build failed: Error: No PostCSS Config found in: /home/rknaur/react-ui-components/dist
    at /home/rknaur/manageiq-ui-classic/node_modules/postcss-load-config/index.js:51:26
    at <anonymous>
 @ ../react-ui-components/dist/tagging.css 4:14-324
 @ ./app/javascript/components/taggingWrapper.jsx
 @ ./app/javascript/packs/component-definitions-common.js

ERROR in ../react-ui-components/dist/tagging.css
Module build failed: ModuleBuildError: Module build failed: Error: No PostCSS Config found in: /home/rknaur/react-ui-components/dist
    at /home/rknaur/manageiq-ui-classic/node_modules/postcss-load-config/index.js:51:26
    at <anonymous>
    at runLoaders (/home/rknaur/manageiq-ui-classic/node_modules/webpack/lib/NormalModule.js:193:19)
    at /home/rknaur/manageiq-ui-classic/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /home/rknaur/manageiq-ui-classic/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/home/rknaur/manageiq-ui-classic/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /home/rknaur/manageiq-ui-classic/node_modules/postcss-loader/index.js:148:13
    at <anonymous>

```

Reply:
Ok then